notebooks/demo/mg_jaccard.ipynb

```diff
@@ -0,0 +1,384 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Running Jaccard,Sorensen and Overlap on Multiple GPUs\n",
+    "\n",
+    "This is a Multi-GPU notebook that loads data into a dask_cudf dataframe, create a Graph, and then runs Jaccard, Sorensen and Overlap.\n",
+    "\n",
+    "\n",
+    "\n",
+    "| Author Credit |    Date    |  Update          | cuGraph Version |  Test Hardware        |\n",
+    "|---------------|------------|------------------|-----------------|-----------------------|\n",
+    "| Don Acosta    | 04/21/2023 | created          | 23.06 nightly   |  2xA6000 CUDA 11.7    |\n",
+    "| Brad Rees     | 04/24/2023 | Added RMAT       | 23.06 nightly   |  2xA6000 CUDA 11.7    |\n",
+    "\n",
+    "\n",
+    "CuGraph's multi-GPU features leverage Dask. RAPIDS has other projects based on Dask such as dask-cudf and dask-cuda. These products will also be used in this example. Check out [RAPIDS.ai](https://rapids.ai/) to learn more about these technologies."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Basic setup"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Installing RAPIDS via [Conda or Docker](https://docs.rapids.ai/install#conda) is the easiest way to setup an environment to run cugraph notebooks."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import needed libraries. We recommend using the [cugraph_dev](https://github.com/rapidsai/cugraph/tree/branch-23.02/conda/environments) env through conda\n",
```

Comment:
This is still pointing to 23.03, which probably doesn't matter much, but it might be better to always point to the latest
```suggestion
    "# Import needed libraries. We recommend using a [conda environment](https://github.com/rapidsai/cugraph/tree/HEAD/conda/environments) provided in the cugraph repo.\n",
```